_tags

```diff
@@ -1,4 +1,4 @@
 true: annot, bin_annot, safe_string
-true: warn(A-44)
+true: warn(A-37-44)
```

Comment:
warning 37 is `unused constructor` -- I'd be in favour of not defining unused constructors, rather than disabling the warning

Reply:
@hannesm Addressed in https://github.com/mirage/mirage-solo5/pull/27/commits/61925cfdb20629a4a866123a74d003f73c5921d7 and 2cf18e3 by moving the solo5_result type into OS.Solo5 module